polaris-react/src/components/SkeletonPage/SkeletonPage.scss

```diff
@@ -46,10 +46,12 @@ $skeleton-display-text-max-width: 120px;
 
 .Title {
   @include text-emphasis-strong;
+  // Question
   font-size: 24px;
   line-height: 28px;
 
   @include when-typography-not-condensed {
+    // Question
     font-size: 20px;
```

Comment:
I noticed some values in `SkeletonPage` that referenced hard coded typography values. Is there a reason for this (maybe because of what styles it can access at that point in the render?). I also realize we might be deprecating `SkeletonPage` in favor of more specific skeleton states as part of the perceived performance project?

Reply:
Going to remove the comments here for merging...but still wonder about this.